queries/blueprint/highlights.scm

```diff
@@ -0,0 +1,57 @@
+(object_id) @variable
+
+(string) @string
+(escape_sequence) @string.escape
+
+(comment) @comment
+
+(constant) @constant.builtin
+
+(boolean) @boolean
+
+(using) @include
+
+(template) @keyword
+
+(decorator) @attribute
+
+(property_definition (property_name) @parameter)
```

Comment:
I don't know the exact context here. But might `@property` be more appropriate here?

Reply:
Sure, that's more appropriate. Just changed it.